addOns/pscanrulesBeta/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+### Added
+- Polyfill.io script detection
```

Comment:
cdn or domain instead of "script"?

Reply:
Well, its detecting potential scripts on the polyfill.io domain, thats what I meant..